akka-actor-typed/src/main/scala/akka/actor/typed/ActorRef.scala

```diff
@@ -134,3 +140,13 @@ object RecipientRef {
     def !(msg: T): Unit = ref.tell(msg)
   }
 }
+
+/**
+ * INTERNAL API
+ *
+ * An ActorRef that ignores any incoming messages.
+ * This reference cannot watch other references.
+ *
+ */
+@InternalApi private[typed] final class IgnoreActorRef[T]()
+    extends ActorRefAdapter[T](new classic.IgnoreActorRef("local"))
```

Comment:
I wonder if the class is needed, or if it would work with a:
```
private val ignoreRef = ActorRefAdapter[Any](new classic.IgnoreActorRef("local"))
```
and 
```
def ignore[T]: ActorRef[T] = ignoreRef.unsafeCast[T] // safe cast
```